columnflow/production/normalization.py

```diff
@@ -197,6 +202,10 @@ def normalization_weights(self: Producer, events: ak.Array, **kwargs) -> ak.Arra
     norm_weight = events.mc_weight * process_weight
     events = set_ak_column(events, self.weight_name, norm_weight, value_type=np.float32)
 
+    # If we are stitching, we also compute the inclusive weight for debugging purposes
+    if self.allow_stitching and self.dataset_inst == self.get_inclusive_dataset():
```

Comment:
Mh, could you store the reference to the inclusive dataset on `self` in `init`, and use it here?